src/lib.rs

```diff
@@ -0,0 +1,3 @@
+extern crate ykstackmaps;
```

Comment:
Is that required in Rust 2018? And at least one other extern site.

Reply:
Good point. Yes, we should use the 2018 edition unless there's a compelling reason not to.

Reply:
He already has `edition = "2018"`, so I'm surprised it didn't complain. Sure you didn't miss warnings Jake?

Reply:
Hmm, no warnings but I've removed the duplicate extern site as that shouldn't have been there.